### ISSUE TITLE
`savemat` creates files incompatible with MATLAB on some locales

### ISSUE THREAD

**Issue by BSchilperoort** [frejanordsiek/hdf5storage#123]
When using `savemat` to write a .mat file, with a system locale such as Chinese, savemat writes a header containing Chinese characters. MATLAB is not able to read these files and fails the open the .mat file:

`Error using load - Unable to read MAT-file example.mat. Not a binary MAT-file. Try load -ASCII to read as text.`

The offending code is here;
https://github.com/frejanordsiek/hdf5storage/blob/ec2e1e34789ad27713b860eb4de5da99817fa315/hdf5storage/__init__.py#L1711

As `strftime` uses the locale to determine the month and day names, these can contain characters that MATLAB cannot read;

```
MATLAB 7.3 MAT-file, Platform: CPython 3.8.13, Created on: 周四 8月 25 15:40:44 2022 HDF5 schema 1.00 .
```

Interestingly, Octave seems to be fine with the Chinese characters 🤷 

<hr>

Probably related to #116

Our discussion: https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/issues/38

**Comment by frejanordsiek** [comment]
Fixed in commit 3d54b5624d6974c95ddc6a791257276072e441f5 in the 0.1.x branch. This package now forces the use of English month and day of the week names, regardless of locale.

**Comment by BSchilperoort** [comment]
Nice, thanks for fixing this!
